src/llscan.cc

```diff
@@ -241,7 +233,19 @@ bool FindInstancesCmd::DoExecute(SBDebugger d, char** cmd,
     }
 
   } else {
-    result.Printf("No objects found with type name %s\n", type_name.c_str());
+    Settings* settings = Settings::GetSettings();
+    if (settings->ShouldUseColor()) {
+      rang::setControlMode(rang::control::Force);
+    } else {
+      rang::setControlMode(rang::control::Off);
+    }
+
+    std::stringstream ss;
+    ss << rang::style::bold << rang::fg::red
+       << "No objects found with type name " << type_name.c_str()
```

Comment:
I think `<< type_name` (without the `.c_str()`) will work, here and in other stringstreams.